guides/micronaut-aws-lambda-eventbridge-event/micronaut-aws-lambda-eventbridge-event.adoc

```diff
@@ -22,7 +22,7 @@ If you use https://launch.micronaut.io[Micronaut Launch], select serverless func
 
 common:default-package.adoc[]
 
-common:annotationprocessors.adoc[]
+https://guides.micronaut.io/latest/micronaut-intellij-idea-ide-setup.html[Setup IntelliJ IDEA to develop Micronaut Applications].
```

Comment:
suggestion: Could be worth setting this as a property in the asciidoctor build properties, and use that property in the docs. If we have to update the URL there would be a single place to change.